src/components/landing-layout/footer/language-picker.tsx

```diff
@@ -65,8 +74,7 @@ const LanguagePicker: React.FC = () => {
       size="sm"
       className="mb-2 mx-auto w-auto"
       value={ findLanguageCode(i18n.language) }
-      onChange={ onChangeLang() }
-    >
+      onChange={ onChangeLang() }>
```

Comment:
Any reason for the additional brackets?

```suggestion
      onChange={ onChangeLang }>
```

Reply:
fixed by refactoring